### ISSUE TITLE
Fullscreen tiling using built in full screen support (primary / auxiliary)

### ISSUE THREAD

**Issue by gigq** [kasper/phoenix#345]
- Version: 4.0.0
- macOS: 14.3.1

I see it is possible to set a window to full screen but I can't figure out if there is any way to set a window to be tiled with another window fullscreen. The Apple docs mention this as full screen primary and full screen auxiliary, but even in their docs I can't find the method to set a window to be primary or auxiliary. Is there just no API for this from Apple or is it just a feature that hasn't been added to Phoenix yet.

**Comment by kasper** [comment]
@gigq Hey! So basically a split full screen with two windows? I haven’t seen an API for that, I will need to dig a bit. Might be that the APIs are private.

**Comment by gigq** [comment]
Yes that’s what I’m looking for. But I haven’t been able to find any software that does it so you might be right.
